ferretdb/ferretdb_test.go

```diff
@@ -85,3 +90,52 @@ func Example_unix() {
 
 	// Output: mongodb://%2Ftmp%2Fferretdb-27017.sock
 }
+
+func Example_tls() {
+	certPath := path.Join("..", "build", "certs", "server-cert.pem")
+	keyPath := path.Join("..", "build", "certs", "server-key.pem")
+
+	f, err := New(&Config{
+		Listener: ListenerConfig{
+			TLS:         "127.0.0.1:47017",
+			TLSCertFile: certPath,
+			TLSKeyFile:  keyPath,
+		},
+		Handler:       "pg",
+		PostgreSQLURL: "postgres://postgres@127.0.0.1:5432/ferretdb",
+	})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	done := make(chan struct{})
+
+	go func() {
+		log.Print(f.Run(ctx))
+		close(done)
+	}()
+
+	uri := f.MongoDBURI()
+	fmt.Println(uri)
+
+	// Use MongoDB URI as usual. To connect to TLS listener, set TLS config.
+	// For example:
+	//
+	// import "go.mongodb.org/mongo-driver/mongo"
+	// import "go.mongodb.org/mongo-driver/mongo/options"
+	//
+	// [...]
+	//
+	// 	tlsConfig, err := options.BuildTLSConfig(map[string]interface{}{
+	//		"tlsCAFile": path.Join("path", "to", "rootCA.pem"),
```

Comment:
That's only needed for custom CAs. Let's either omit that completely, or mention that in the text